content/docs/use-cases/experiment-bookkeeping.md

```diff
@@ -0,0 +1,32 @@
+# Data Science Experiment Bookkeeping
+
+Mature data science required constant research and improvement. This implies
```

Comment:
DS is a highly iterative process where DSs and ML engineers tend to run a lot of attempts (aka experiments, run, etc) by tuning paramers, data, code and collecting metrics. Being able to track, compare, visualize, share metrics. Being able to connect those back to the setup (params, data, ...) of an experiment. Being able to save and share an experiments in a reproducible fashion. Being able to "productionize" the result when time comes. Those tasks are an essential part for the ML infra for efficient teams ....


please combine/take some ideas/terms to improve the current version which is a bit too abstract a the moment. 

but I also agree that we should keep it high level

Reply:
OK will do.

I just wanted to keep this very short and go straight to the problem/solution story instead of repeating the intro from https://dvc.org/doc/user-guide/experiment-management.